src/server/page_server.cpp

```diff
@@ -485,12 +492,14 @@ page_server::set_passive_tran_server_connection (cubcomm::channel &&chn)
 
   chn.set_channel_name ("PTS_PS_comm");
 
+  assert (chn.is_connection_alive ());
   const auto channel_id = chn.get_channel_id ();
 
   er_log_debug (ARG_FILE_LINE, "Passive transaction server connected to this page server. Channel id: %s.\n",
 		channel_id.c_str ());
 
-  m_passive_tran_server_conn.emplace_back (new connection_handler (chn, transaction_server_type::PASSIVE, *this));
+  m_passive_tran_server_conn.emplace_back (new connection_handler (chn, transaction_server_type::PASSIVE, *this,
```

Comment:
I don't we need to pass the channel id explicitly. the connection_handler already got the channel and we can just use ch.get_channel_id() in the contr of the connection_handler.

Reply:
Oh, OK. I understand.
We can just pass the channel (`chn`), so we don't need to pass the channel ID.